brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -51,6 +51,11 @@ std::vector<hsize_t> _computeChunkDims(const std::vector<uint32_t>& cellSizes,
                                        float cellsToFramesRatio,
                                        size_t blockSize = _blockSize)
 {
+    if (cellsToFramesRatio == 0)
```

Comment:
I typically use integers and rely on the compiler to do the right thing. But it may be a bad practice on my side.
I've added the check in the parsing code, so this function will never get a negative number. Do you want an assert?

Reply:
It is fine, you can leave it as is.